NDB.Covid19/NDB.Covid19.iOS/Views/AuthenticationFlow/CountriesConsentViewController.cs

```diff
@@ -29,7 +31,34 @@ public override void ViewDidLoad()
 
             SetTexts();
             SetupStyling();
-            SetupAccessibility();  
+            SetupAccessibility();
+            AddObservers();
+        }
+
+        public override void ViewWillAppear(bool animated)
+        {
+            base.ViewWillAppear(animated);
+            LogUtils.LogMessage(LogSeverity.INFO, "The user is seeing Countries Consent", null, GetCorrelationId());
+        }
+
+        public override void ViewWillDisappear(bool animated)
+        {
+            base.ViewWillDisappear(animated);
+            MessagingCenter.Unsubscribe<object>(this, MessagingCenterKeys.KEY_APP_RETURNS_FROM_BACKGROUND);
+            MessagingCenter.Unsubscribe<object>(this, MessagingCenterKeys.KEY_APP_WILL_ENTER_BACKGROUND);
+        }
+
+        void AddObservers()
+        {
+            MessagingCenter.Subscribe<object>(this, MessagingCenterKeys.KEY_APP_RETURNS_FROM_BACKGROUND, (object _) =>
+            {
+                LogUtils.LogMessage(LogSeverity.INFO, "The user is seeing Countries Consent", null, GetCorrelationId());
+            });
+
+            MessagingCenter.Subscribe<object>(this, MessagingCenterKeys.KEY_APP_WILL_ENTER_BACKGROUND, (object _) =>
+            {
+                LogUtils.LogMessage(LogSeverity.INFO, "The user is left Countries Consent", null, GetCorrelationId());
```

Comment:
Check out what's the log statement in Android. I guess more correct would be "has left" or just "left"

Reply:
If have corrected the log message, so that it is now identical to the Android message.